src/Integration.Vsix/SolutionAnalysisRequester.cs

```diff
@@ -0,0 +1,106 @@
+//-----------------------------------------------------------------------
+// <copyright file="SolutionAnalysisRequester.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Options;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace SonarLint.VisualStudio.Integration.Vsix
+{
+    internal class SolutionAnalysisRequester
```

Comment:
Good question. If we don't do anything, then we'll have the same functionality as today with version 2.1. So it's not such a big deal. But we could notify the user that something is wrong, and he might have to restart VS after the bind.